isat/pkg/storage/local/local.py

```diff
@@ -0,0 +1,13 @@
+import os
+from isat.pkg.models.image import Image
+
+
+class Local_Storage:
+    def __init__(self, directory_path):
+        self.directory = directory_path
+        if os.path.exists(self.directory) is False:
```

Comment:
`if os.path.exists(self.directory)` is enough, direct comparison with false is redundant.